src/views/SignupAfterMailView.vue

```diff
@@ -15,21 +15,22 @@ function onResendEmail() {
 
 <template>
   <div
-    class="flex items-center justify-center bg-background-tertiary px-8 py-6"
+    class="flex flex-col items-center bg-background-tertiary"
     style="height: calc(100vh - 56px)"
   >
-    <div class="max-w-xl space-y-5 rounded-2xl bg-white px-14 py-10">
-      <div class="fontstyle-ui-title text-left">確認メールを送信しました</div>
-      <div class="fontstyle-ui-body text-left text-text-secondary">
+    <div class="w-90 space-y-6 rounded-2xl bg-white px-8 py-6">
+      <div class="fontstyle-ui-subtitle text-left text-text-primary">確認メールを送信しました</div>
+      <div class="fontstyle-ui-body text-left text-text-primary">
```

Comment:
```suggestion
      <div class="fontstyle-ui-body text-text-primary">
```
ここのtext-leftは不要な気がします。 (SignupView, SignupRegisterView, SignupAfterMailViewにいくつかあります)